roles/edpm_update/molecule/default/Containerfile.j2

```diff
@@ -5,9 +5,9 @@ FROM {{ item.registry.url }}/{{ item.image }}
 FROM {{ item.image }}
 {% endif %}
 
-RUN curl -o /etc/yum.repos.d/delorean.repo https://trunk.rdoproject.org/centos9-wallaby/current/delorean.repo && \
```

Comment:
please explain this change here. shouldn't we test edpm_update using wallaby repos?